rel/vm.args.eex

```diff
@@ -0,0 +1,14 @@
+## Customize flags given to the VM: http://erlang.org/doc/man/erl.html
```

Comment:
this is a stock `vm.args.eex`, maybe we should just remove it. Or keep it, in case we ever need to put something here? (though, in that case we could just run `mix release.init` again or just create it from scratch)

Reply:
My plan is to play around with this stuff [1] so we can keep it for now.

https://elixirforum.com/t/performance-of-erlang-elixir-in-docker-kubernetes/21493/18